definitions/CMakeLists.txt

```diff
@@ -16,10 +16,17 @@
 # limitations under the License.
 find_package(JsonGenerator REQUIRED)
 
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Please raise the minimum CMake version at the top-level CMakeLists.txt found here:
https://github.com/rdkcentral/ThunderInterfaces/blob/master/CMakeLists.txt#L18

Reply:
Done